features/netsocket/cellular/generic_modem_driver/TESTS/unit_tests/default/template_mbed_app.txt

```diff
@@ -0,0 +1,69 @@
+{
```

Comment:
what's the intention with this .txt ? It is just a template for json file, that is not present but required by these tests. I dont see any defaults here?